std/convert/markdown_test.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+import { assert, assertEquals } from "../testing/asserts.ts";
+import { test } from "../testing/mod.ts";
+import { MarkdownOptions, parse } from "./markdown.ts";
+
+const FIXTURE_DIR = "./std/convert/fixtures";
+
+const inputs = Deno.readDirSync(FIXTURE_DIR);
+const decoder = new TextDecoder();
+
+function parseOptions(str: string): [string, Partial<MarkdownOptions> | undefined] {
```

Comment:
I guess this function parses the front-matter of markdown. Maybe we should have another module for this later. (I think it's ok to have this here for now.)

Reply:
This is specific to the fixtures I ported from marked, I didn't want to change them.